deploy/resources/v0.8.0/addons/clustertasks/buildah/buildah-task-snippet.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleYAMLSample
+metadata:
+  name: buildah-snippet
+spec:
+  targetResource:
+    apiVersion: tekton.dev/v1alpha1
+    kind: Pipeline
+  title: buildah task
+  description: An example of buildah build task
```

Comment:
`description` should describe what the task does rather than what the snippet is. This text will be displayed under the title to give user enough information about the task


<img width="351" alt="snippet" src="https://user-images.githubusercontent.com/1174934/70345548-53b0a500-182a-11ea-8df5-cbde2bed7ff4.png">

Reply:
Same applies to the rest of snippets in this PR